src/types/statement.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+export type SimpleStatement = {
```

Comment:
hmm, that's a good point.  It would be nice not to have to manually keep these in sync